src/Command/JobCommand.php

```diff
@@ -69,7 +69,6 @@ public function templateData(Arguments $arguments): array
      * Gets the option parser instance and configures it.
      *
      * @param \Cake\Console\ConsoleOptionParser $parser The parser to update.
-     * @return \Cake\Console\ConsoleOptionParser
```

Comment:
I think we need these type in docblocks for API docs generation. Right @othercorey ?